projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/methods-and-properties.migration.ts

```diff
@@ -0,0 +1,1790 @@
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const METHODS_AND_PROPERTIES_DEPRECATIONS_DATA: MethodPropertyDeprecation[] =
+  [
+    {
+      class: 'QuickOrderFormComponent',
+      importPath: '@spartacus/cart/quick-order/components',
+      deprecatedNode: 'globalMessageService',
+      comment:
+        "// TODO:Spartacus - Property 'globalMessageService' was removed from Class 'QuickOrderFormComponent'. ",
+    },
+    {
+      class: 'QuickOrderService',
+      importPath: '@spartacus/cart/quick-order/core',
+      deprecatedNode: 'productAdapter',
+      comment:
+        "// TODO:Spartacus - Property 'productAdapter' was removed from Class 'QuickOrderService'. ",
+    },
+    {
+      class: 'QuickOrderService',
+      importPath: '@spartacus/cart/quick-order/core',
+      deprecatedNode: 'removeEntry',
+      comment:
+        "// TODO:Spartacus - Method 'removeEntry' was removed from Class 'QuickOrderService'. Use 'softDeleteEntry' instead.",
+    },
+    {
+      class: 'QuickOrderService',
+      importPath: '@spartacus/cart/quick-order/core',
+      deprecatedNode: 'search',
+      comment:
+        "// TODO:Spartacus - Method 'search' was removed from Class 'QuickOrderService'. Use 'searchProducts' instead.",
+    },
+    {
+      class: 'QuickOrderFacade',
+      importPath: '@spartacus/cart/quick-order/root',
+      deprecatedNode: 'removeEntry',
+      comment:
+        "// TODO:Spartacus - Method 'removeEntry' was removed from Class 'QuickOrderFacade'. ",
+    },
+    {
+      class: 'QuickOrderFacade',
+      importPath: '@spartacus/cart/quick-order/root',
+      deprecatedNode: 'search',
+      comment:
+        "// TODO:Spartacus - Method 'search' was removed from Class 'QuickOrderFacade'. ",
+    },
+    {
+      class: 'AddToSavedCartComponent',
+      importPath: '@spartacus/cart/saved-cart/components',
+      deprecatedNode: 'activeCartService',
+      comment:
+        "// TODO:Spartacus - Property 'activeCartService' was removed from Class 'AddToSavedCartComponent'. ",
+    },
+    {
+      class: 'SavedCartAdapter',
+      importPath: '@spartacus/cart/saved-cart/core',
+      deprecatedNode: 'saveCart',
```

Comment:
NOT related, but I always forget to ask

don't have to answer, but I'll leave it here for me to remember

Why did this move to the cart adapter as save cart is part of the Saved Cart controller in the backend, so I would guess to look at the saved cart domain in the frontend

Reply:
function `saveCart` is not only used in SavedCart domain, it also used in other places. So, that's why we move this function to `base`.